File: test-sources/test-sigfpe.c
/* $Id$
 *
 * This should fail with RUN-ERROR due to integer division by zero,
 * giving an exitcode 136.
 */

#include <stdio.h>

int main()
{
  int a = 0;
  int b;

  b = 10 / a;

  printf("%d\n",b);

  return 0;
}

File: test-sources/test-sigfpe.java
/* $Id$
 *
 * This should fail with RUN-ERROR due to integer division by zero.
 */

import java.io.*;

class Main {
    public static void main(String[] args) throws Exception {
		int a = 0;
		int b = 10 / a;
		System.out.println(b);
    }
}
